vcf_annotation_tools/vcf_info_annotator.py

```diff
@@ -74,6 +80,9 @@ def define_parser():
         "-s", "--source",
         help="The string to put in the \"source\" section of the INFO header line - optional "
     )
+    parser.add_argument('-w', action='store_true',
```

Comment:
What do you think of adding a long argument name here as well?

Reply:
I tried that and it yelled at me :)  Let me try again